.circleci/config.yml

```diff
@@ -72,7 +75,38 @@ jobs:
           - run:
               name: Ship Build File To S3 Bucket
               command: |
-                 cd TerraWeb && aws s3 mv ./build s3://${BucketName} --recursive --acl public-read-write
+                 aws configure set aws_access_key_id $adminAccessKey
+                 aws configure set aws_secret_access_key $adminSecretAccessKey
+                 aws configure set default.region us-east-1
+                 cd LawnNannyWeb
+                 echo $FrontendDeploymentS3
+                 aws s3 mv ./build s3://${FrontendDeploymentS3} --recursive --acl public-read-write
```

Comment:
Is there any way these commands can be moved to a bash script or something similar?

Reply:
I can take a look.